compiler/il/OMRBlock.hpp

```diff
@@ -483,6 +486,7 @@ class OMR_EXTENSIBLE Block : public TR::CFGNode
     */
 
    private:
+   void init(TR::TreeTop *entry, TR::TreeTop *exit);
 
    void uncommonNodesBetweenBlocks(TR::Compilation *, TR::Block *, TR::ResolvedMethodSymbol *methodSymbol = NULL);
```

Comment:
`uncommonNodesBetweenBlocks()` is now private, but it used to be public. Was this change intentional? I would have expected this to break something.

Reply:
I didn't make `uncommonNodesBetweenBlocks` private. It is private on [master](https://github.com/eclipse/omr/blob/master/compiler/il/OMRBlock.hpp#L485) already.

Reply:
Oh, you're right. The diff for this is a bit confusing.